### PR TITLE
Gui improvements

### PR DESCRIPTION
**Pull request by stevenmeijer**
Improves the gui by adding a set of smooth animations to the menu and other windows. 
Splits the ui up into two parts: pre and live production.
Adds a settings menu that controls some settings used by the application.
Removes the unused menu bar.
Adds a loading gif that can be used to indicate a loading part of the application

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/gui/Animation.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+     * Sets a timeout to prevent the window from being changed when a fade out
+     * animation is still playing. Only called from {@link #animNodeOut(Node, boolean)}
+     * 
+     * @param dur The duration of the timeout.
+     */
+    private static void animTimeout(int dur) {
+        Timeline timeout = 
+                new Timeline(
+                        new KeyFrame(
+                                Duration.millis(dur)));
```
Comment (tkolenbrander) [inline src/main/java/nl/tudelft/contextproject/gui/Animation.java]: Can't you just put this on one line? Looks a bit weird right now.

**Comment by tkolenbrander** [comment]
When playing around with the interface I noticed a delay when opening one of the live views or the preview interface. However, this probably isn't your fault. These interfaces just take longer to load. 

Your code looks great overall and I really like the animations. Especially the button/image used for the settings menu looks good.

File: src/main/java/nl/tudelft/contextproject/gui/Animation.java (hunk 1 above)
Comment (stevenmeijer) [inline src/main/java/nl/tudelft/contextproject/gui/Animation.java]: You're correct

**Comment by tkolenbrander** [comment]
Thank you for changing that line, I think we can merge your branch.

**Comment by TimYuenior** [comment]
Nothing to add here. You have my vote for a merge.

**Comment by bvobart** [comment]
Looks good, both in code quality and in GUI. Nice work on putting those animations in that utility class. The settings button does not work as of yet, but that is no problem as it needs the ApplicationSettings class from the bugFix branch, which is why we have decided to work on that in the next sprint. I'll accept this PR now.

**Pull request merged**